views/js/pciCreator/dev/audioRecordingInteraction/runtime/js/recorder.js

```diff
@@ -107,6 +118,39 @@ define([
             analyser,               // the WebAudio node use to read the input level
             frequencyArray;         // used to compute the input level from the current array of frequencies
 
+        /**
+         * Check if client uses the iOS device.
+         *
+         * @param {Array} frequencyArray
+         * @returns {*|boolean}
+         */
+        function isIOSDevice(frequencyArray) {
```

Comment:
u dont need anymore to pass the frequencyArray

Reply:
Done.